site/gatsby-site/src/components/SemanticallyRelatedIncidents.js

```diff
@@ -45,11 +46,18 @@ const SemanticallyRelatedIncidents = ({ incident, editable }) => {
 
   const [error, setError] = useState(null);
 
+  const initialDisplay = useRef(true);
```

Comment:
Why use a ref here instead of a state variable?

Reply:
This just tracks whether we're on the first render, so it shouldn't trigger a re-render.

Reply:
It is not very "reacty", but ok.